src/es_sfgtools/processing/assets/tiledb_temp.py

```diff
@@ -0,0 +1,188 @@
+import tiledb
+import numpy as np
+from pathlib import Path
+from datetime import datetime
+import pandas as pd
+from pandera import check_types
+from pandera.typing import DataFrame
+
+from typing import Optional,Dict
+
+from .observables import AcousticDataFrame,GNSSDataFrame,PositionDataFrame,ShotDataFrame
+
+filters = tiledb.FilterList([tiledb.ZstdFilter(5)])
+TimeDomain = tiledb.Dim(name="time", domain=(0,), tile=100, dtype="datetime64['ms']")
+attribute_dict: Dict[str,tiledb.Attr] = {
+    "east": tiledb.Attr(name="east", dtype=np.float32),
+    "north": tiledb.Attr(name="north", dtype=np.float32),
+    "up": tiledb.Attr(name="up", dtype=np.float32),
+    "east_std": tiledb.Attr(name="east_std", dtype=np.float32,nullable=True),
+    "north_std": tiledb.Attr(name="north_std", dtype=np.float32,nullable=True),
+    "up_std": tiledb.Attr(name="up_std", dtype=np.float32,nullable=True),
+    "latitude": tiledb.Attr(name="latitude", dtype=np.float32),
+    "longitude": tiledb.Attr(name="longitude", dtype=np.float32),
+    "height": tiledb.Attr(name="height", dtype=np.float32)
+}
+
+GNSSAttributes = [
+    attribute_dict["latitude"],
+    attribute_dict["longitude"],
+    attribute_dict["height"],
+    attribute_dict["east"],
+    attribute_dict["north"],
+    attribute_dict["up"],
+    attribute_dict["east_std"],
+    attribute_dict["north_std"],
+    attribute_dict["up_std"],
+    tiledb.Attr(name="number_of_satellites", dtype="uint8"),
+    tiledb.Attr(name="pdop", dtype=np.float32),
+]
```

Comment:
we should also parse the per constellation number of satellite data, so have all of these attributes "Nsat","G","R","E","C2","C3","J"

Reply:
and include the uncertainty `wrms` here as well (maybe as nullable since it comes from a different file)